samtranslator/model/apigatewayv2.py

```diff
@@ -17,6 +17,7 @@ class ApiGatewayV2HttpApi(Resource):
         "BodyS3Location": PropertyType(False, is_type(dict)),
         "Description": PropertyType(False, is_str()),
         "FailOnWarnings": PropertyType(False, is_type(bool)),
+        "Name": PropertyType(False, is_str()),
```

Comment:
We should use `Property` and make this any type since it's pass-through.

Reply:
changed